src/parser/parser.ts

```diff
@@ -13,7 +13,9 @@ import { OrganizationalUnitResource } from './model/organizational-unit-resource
 import { yamlParse } from '~yaml-cfn/index';
 import { FileUtil } from '~util/file-util';
 import { yamlParseContentWithIncludes } from '~yaml-cfn/yaml-parse-includes';
+import { nunjucksParseContentWithIncludes } from '~yaml-cfn/nunjucks-parse-includes';
 
+const NUNJUCKS_FILE_EXTENSION = '.nj';
```

Comment:
nunjucks community recommends `.njk`, https://mozilla.github.io/nunjucks/templating.html#file-extensions

Reply:
is mistaking the right file extension not also an argument to not tie text-templating to the extension of the template?

Reply:
removed trigger of templating by filename